instrumentation/redis/README.md

```diff
@@ -41,6 +41,18 @@ OpenTelemetry::Instrumentation::Redis.with_attributes('peer.service' => 'cache')
 end
 ```
 
+### Configuration options
+
+```ruby
+OpenTelemetry::SDK.configure do |c|
+  c.use 'OpenTelemetry::Instrumentation::Redis', {
+    # The obfuscation of arguments in the db.statement attribute is disabled by default.
+    # To enable, set enable_arg_obfuscation to true.
+    enable_arg_obfuscation: false,
```

Comment:
It's unfortunate that we chose to call the config option for MySQL and Postgres `enable_sql_obfuscation`. I wonder if we'd be better off renaming those and this option to `enable_statement_obfuscation`, both for consistency and because we're obfuscating the `db.statement` attribute.

Reply:
This should be true by default (and should be true for the others as well), following a "secure by default" mindset.

Reply:
I agree with you both here. Speaking from my own experience it’s better to not leave this as an opt-in as a security incident will soon follow.

I can change it to `disable_statement_obfuscation` if there’s consensus.

Reply:
> I can change it to `disable_statement_obfuscation` if there’s consensus.

Please do 🙏

Reply:
Alternatively, just change the default to `true`.